### PR TITLE
ADD 10.13.0.RC 2

### PR DESCRIPTION
**Pull request by jnweiger**
Adding drawio-1.0.1,
fixing ie11 compatibitlity
and re-re-re-fixing heic

### CODE REVIEW

**Comment by phil-davis** [comment]
Note: that "RC2" tarball still has this in `version.php`
```
$OC_Version = array(10,13,0,1);
$OC_VersionString = '10.13.0 beta 1';
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud-docker/server/1588
`.drone.star:7:18: got string literal, want '}'`

File: .drone.star [hunk 1]
```diff
@@ -1,9 +1,9 @@
 def main(ctx):
     versions = [
         {
-            "value": "10.13.0-rc.1",
-            "tarball": "https://download.owncloud.com/server/testing/owncloud-complete-20230816.tar.bz2",
-            "tarball_sha": "0e92b52bc16540b7f13bcf94870439e8db6406b29cf1a7a9ecd9a58652875f37",
+            "value": "10.13.0-rc.2",
+            "tarball": "https://download.owncloud.com/server/testing/owncloud-complete-20230821.tar.bz2",
+            "tarball_sha": "58b56fed6b0a5a6efdf3d0dbdd30ac7ecd4fea01638b3878d72af8378dac7e60"
```
Comment (phil-davis) [inline .drone.star:6]:
```suggestion
            "tarball_sha": "58b56fed6b0a5a6efdf3d0dbdd30ac7ecd4fea01638b3878d72af8378dac7e60",
```

**Comment by jnweiger** [comment]
> Note: that "RC2" tarball still has this in `version.php`
> 
> ```
> $OC_Version = array(10,13,0,1);
> $OC_VersionString = '10.13.0 beta 1';
> ```

My bad. I need to fix the github permissions. I always expect, that I can push to my branch, but rules are so strict, that the push sometimes fails. Sorry, I'll do a rebuild.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**